server/protocols/webadmin/webadmin-mailbox/src/test/java/org/apache/james/webadmin/routes/DomainQuotaRoutesNoVirtualHostingTest.java

```diff
@@ -44,7 +44,7 @@
 class DomainQuotaRoutesNoVirtualHostingTest {
 
     private static final String QUOTA_DOMAINS = "/quota/domains";
-    private static final String FOUND_COM = "found.com";
+    private static final String FOUND_LOCAL = "found.local";
```

Comment:
I'm late to commentthis review but `.local` doesn't seem wise as it's never considered an internet domain (see https://en.wikipedia.org/wiki/.local )

Reply:
no a big issue imho, but if needed we can use .test instead.